### PR TITLE
Jake smith branch (include on public listing)

### PR DESCRIPTION
**Pull request by SaltstormJake**
# Pull Request Name / Title
Created my own page

# Category
- [x] One-Month Project Submission
- [ ] Bug Fix

# PR Checklist
Complete this checklist before opening your 
- [ ] If the PR is a bug fix, the issue is linked in the PR Content
- [X] If this PR is a One-Month submission, include if you want your site to be listed publicly on our website.
- [ ] If this PR is a work in progress, it's name starts with `WIP:`. Example: `WIP: Nathan Solomon One-Month Submission`

### CODE REVIEW

File: members/jakesmith/index.html [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+                I specialize in C# programming whilst being a fast learner 
+                when it comes to any other language or framework I am required to learn.
+                ​<br>
+                <br>            
+                I am extremely self-motivated, getting fulfillment out of learning new skills. I have historically been known for working well alongside others in groups, and have experience putting together projects remotely.
+            </big></p>
+        </center></summary>
+        <br>
+        <br>
+        <!--<education><center>
```
Comment (Jedimaster4559) [inline members/jakesmith/index.html]: Did you intend to leave so much of this commented out here?

Comment (Jedimaster4559) [reply members/jakesmith/index.html]: I see a lot of this content on other pages actually. Do we need to keep it here as well?

**Review by Jedimaster4559 — DISMISSED** [review]
Looks great! I'm going ahead and approving this! Let me know if you want your project publicly listed on the membership page and I'll try to take care of that tomorrow.

**Review by Duke02 — APPROVED** [review]
Seems decent. Would personally advise against putting emails and phone numbers as they can be scraped and used for spam calls/emails, but nothing that I think is horribly wrong.

**Comment by Jedimaster4559** [comment]
I have to agree with @Duke02. You might want to eventually remove your personal details.

**Review by Jedimaster4559 — APPROVED** [review]

**Review by sel3450 — APPROVED** [review]

**Pull request merged**